waypoints:
  - name: ponto1
    pose:
      x: 5.3
      y: 1.8
      yawn: 0
  - name: ponto2
    pose:
      x: 0.5
      y: 0.6
      yawn: 0.5
  - name: ponto3
    pose:
      x: 5.2
      y: -0.9
      yawn: -0.617
  - name: ponto4
    pose:
      x: 0.5
      y: -2.3
      yawn: -0.617
  - name: corner1
    pose:
      x: 1.5
      y: 0.5
      yawn: -0.999
  - name: corner2 
    pose:
      x: -1.5
      y: 0
      yawn: -0.999
da_wae:
#   - ponto1
#   - ponto2
#   - ponto3
   - ponto4

